src/vs/workbench/services/extensions/common/extensionsRegistry.ts

```diff
@@ -387,6 +387,7 @@ export interface IExtensionPointDescriptor {
 export class ExtensionsRegistryImpl {
 
 	private readonly _extensionPoints = new Map<string, ExtensionPoint<any>>();
+	private _resolvedUsers = new Map<IExtensionDescription, boolean>();
```

Comment:
Does this file have to be the one that gets the changes?

We try to not really touch the src/vs files and keep our work in src/sql. Since the extension contribution point is given to us by our product, can we keep this checking system in our code?